src/visualizers/Chaos.ts

```diff
@@ -128,8 +133,8 @@ class Chaos extends P5Visualizer {
                 + 'at the first valid index.',
         },
         last: {
-            value: '' as string | number,
-            forceType: 'integer',
+            value: 1000,
```

Comment:
Basically the same comment as the last one: this seems to have changed the behavior to initially fill `1000` rather than start empty to mean `use the last index of the sequence`. Many sequences have many more than 1000 terms. Please restore the prior behavior.